### ISSUE TITLE
front_end/tool/perf_test doesn't work with new typedef syntax

### ISSUE THREAD

**Issue by floitschG** [dart-lang/sdk#30825]
/cc @sigmundch 

The `front_end/tool/perf_test` isn't working with the new typedef syntax.

I'm marking it as `fail` in the status file.

**Comment by peter-ahe-google** [comment]
This is an old test based on dartk and should just be deleted. Correct, @sigmundch ?

**Comment by sigmundch** [comment]
correct, we need to first delete the corresponding benchmark in our perf bots, though.

**Comment by sigmundch** [comment]
https://dart-review.googlesource.com/c/sdk/+/9790 deletes the use of the old dartk and rasta from the perf scripts.
